### PR TITLE
UI for Preferences Screen 

### PR DESCRIPTION
**Pull request by milaendo**
screen shot 2018-08-21 at 3 06 33 pm

- tests passing locally 
- path is /admin/preferences 
- hooked up to admin sidebar

Ref: #536

### CODE REVIEW

**Comment by velkymx** [comment]
@milaendo Give the first box a title "Localization" and move the name format box to the bottom. On the Second box capitalize all of the words in the title.

**Comment by velkymx** [comment]
The username and password are not the same as the account settings. It's the login for the server.

**Comment by milaendo** [comment]
screen shot 2018-08-22 at 9 53 55 am

File: routes/web.php [hunk 1]
```diff
@@ -58,6 +58,10 @@
       return view('profile',['title' => 'Dashboard']);
     })->name('profile');
 
+    $this->get('/admin/preferences', function(){
+      return view('preferences',['title' => 'Dashboard']);
```
Comment (velkymx) [inline routes/web.php]: Move this into a controller.

**Comment by velkymx** [comment]
@milaendo remove the Notifications completely from the sidebar.

**Pull request merged**